### PR TITLE
Add "Title" directive to evp_test

### PR DESCRIPTION
**Pull request by richsalz**
Should there also be an Info directive that only does TEST_info ?

### CODE REVIEW

File: test/testutil/driver.c [hunk 1]
```diff
@@ -121,11 +121,12 @@ static void finalize(int success)
...
 
-static const char *test_title = NULL;
+static char *test_title = NULL;
 
 void set_test_title(const char *title)
 {
-    test_title = title;
+    free(test_title);
+    test_title = title == NULL ? NULL : strdup(title);
 }
```
Comment (snhenson) [inline test/testutil/driver.c]: Any reason we can't use OPENSSL_strdup/OPENSSL_free?

Comment (richsalz) [inline test/testutil/driver.c]: yes, we want to keep the framework independant from openssl libraries.

Comment (snhenson) [inline test/testutil/driver.c]: Just the framework or tests too? There are quite a few cases where we use OPENSSL_malloc in individual tests.

Comment (richsalz) [inline test/testutil/driver.c]: Just the framework.

**Pull request closed without merging**